opensearch/src/http/transport.rs

```diff
@@ -49,6 +49,7 @@ use crate::{
 };
 use base64::write::EncoderWriter as Base64Encoder;
 use bytes::BytesMut;
+use dyn_clone::clone_trait_object;
```

Comment:
Is this used anywhere?

Reply:
Yeah, it's used at line 493: https://github.com/opensearch-project/opensearch-rs/blob/35da3730f719b9b4b8ccdf5336c8943561725cba/opensearch/src/http/transport.rs#L488-L493